puppet/zulip/manifests/profile/base.pp

```diff
@@ -39,8 +39,9 @@
         'python3-yaml',
         'puppet',
         'git',
-        # Used in scripts including install-yarn.sh
+        # Used for most downloads
         'curl',
+        # Used in check_website_response.sh
         'wget',
```

Comment:
Is there a reason we use `check_website_response.sh` in one place, rather than the standard `check_http` plugin that we use everywhere else?

Reply:
I think it's just history; I suspect in like 2012, there wasn't a `nagios_plugins` package in the version of Debian/Ubuntu being used, or something.